systemtest/src/test/java/io/strimzi/systemtest/kafka/TieredStorageST.java

```diff
@@ -138,10 +147,54 @@ void testTieredStorageWithAivenPlugin() {
         resourceManager.createResourceWithWait(clients.producerStrimzi());
 
         SetupMinio.waitForDataInMinio(suiteStorage.getNamespaceName(), BUCKET_NAME);
+
+        // Create admin-client to check offsets
+        resourceManager.createResourceWithWait(
+            AdminClientTemplates.plainAdminClient(
+                testStorage.getNamespaceName(),
+                testStorage.getAdminName(),
+                KafkaResources.plainBootstrapAddress(testStorage.getClusterName())
+            )
+            .editSpec()
+                .editOrNewTemplate()
+                    .editSpec()
+                        .editFirstContainer()
+                            // TODO - remove this when new version of clients will be available
+                            .withImage("quay.io/strimzi-test-clients/test-clients:latest-kafka-3.7.1")
+                        .endContainer()
+                    .endSpec()
+                .endTemplate()
+            .endSpec().build()
+        );
+        final AdminClient adminClient = AdminClientUtils.getConfiguredAdminClient(testStorage.getNamespaceName(), testStorage.getAdminName());
+
+        TestUtils.waitFor("earliest-local offset to be higher than 0",
+            TestConstants.GLOBAL_POLL_INTERVAL_5_SECS, TestConstants.GLOBAL_TIMEOUT_LONG,
+            () -> {
+                // Fetch earliest-local offsets
+                // Check that data are not present locally, earliest-local offset should be higher than 0
+                String offsetData = adminClient.fetchOffsets(testStorage.getTopicName(), "earliest-local");
```

Comment:
Maybe it would be useful to have some enum with the possible "time options"? In case that we would like to use it somewhere else.

Reply:
I am not against it, but it will produce enum with 5/10 options and only one will be used or enum with just one option. Or we can use this https://github.com/apache/kafka/blob/trunk/clients/src/main/java/org/apache/kafka/common/requests/ListOffsetsRequest.java#L41-L50 and franfesr it to string.

Reply:
Yeah whatever makes sense. I mean, for now we can have just enum with few of the options (or one for the start) and add more in case of need. Or if you don't want to keep it in our STs, we can use the `ListOffsetsRequest` from the Kafka as you mentioned above.

Reply:
Check the changes if you are fine with it now (I ran it against OCP and it worked fine, I will trigger packit when you agree on the solution:) )